src/server-code/main-server.js

```diff
@@ -159,9 +158,12 @@ export default function runMainServer(serverFalcorModel) {
       ' the contents of our lovely newspaper.'
     );
   });
-
+  app.get('/robots.txt', (req, res) => {
```

Comment:
This will cause The Gazelle not to be able to be found in a Google search in general ;). Remember to put it in the `isStaging` conditional!

And also the same small comments as in the Admin server + it's nicer coding style to put the closing parenthesis right after the closing curly bracket, I'm actually surprised the linter doesn't catch this, I guess it's not so opinionated!

So
```
});
```

instead of 
```
}
);
```